src/sessions.rs

```diff
@@ -1,12 +1,13 @@
 use std::os::unix::fs::FileTypeExt;
 use std::{fs, io, process};
 use zellij_utils::{
+    async_std::io::ErrorKind,
```

Comment:
Why can't we use `std::io::ErrorKind` instead of this?

Reply:
No reason, fixed